includes/admin.form.inc

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/**
+ * @file
+ * Admin form for this module.
+ */
+
+/**
+ * Admin form for presbydora.
+ *
+ * @param array $form
+ *   Form object.
+ * @param array $form_state
+ *   Form values.
+ *
+ * @return array
+ *   A renderable array available as a multi-select box.
+ */
+function presbydora_admin_form(array $form, array &$form_state) {
+  $tableselect = presbydora_build_tableselect();
+  $form['presbydora_access_bypass_pids'] = array(
+    '#type' => 'tableselect',
+    '#header' => $tableselect['header'],
+    '#options' => $tableselect['rows'],
+    '#multiple' => TRUE,
+    '#default_value' => variable_get('presbydora_access_bypass_pids', ''),
+  );
+
+  $form['presbydora_submit'] = array(
+    '#type' => 'submit',
+    '#value' => t('Save Configuration'),
+  );
+
+  return $form;
+}
+
+/**
+ * Submit handler for wrlcdora admin form.
+ *
+ * @param array $form
+ *   Form object.
+ * @param array $form_state
+ *   Form values.
+ */
+function presbydora_admin_form_submit(array $form, array $form_state) {
+  module_load_include('inc', 'islandora', 'includes/utilities');
+  $values = $form_state['values'];
+  $pids = $values['presbydora_access_bypass_pids'];
+  if (!empty($pids)) {
+    foreach ($pids as $pid) {
+      if (!islandora_is_valid_pid($pid)) {
+        drupal_set_message(t('@pid is not a valid PID!', array('@pid' => $pid)), 'error');
+      }
+    }
+  }
```

Comment:
Understandable. Removing as it also causing a error to appear on the page when a user deselects everything. "0 is not a valid PID!"

Reply:
Actually... that might be something else... been a while since I've implemented something with a tableselect personally, but... is the array actually an associative array?... as in: mapping the PID to:
* the PID for the items selected; or, 
* zero where something is _not_ selected...

... if this is the case, could remove the zero'd items with an `array_filter()` call.

Reply:
Debugged: you are correct, we are dealing with an assoc array.